Resources/public/js/views/ez-fieldeditview.js

```diff
@@ -130,95 +141,71 @@ YUI.add('ez-fieldeditview', function (Y) {
             }
-            tooltip.addClass(IS_VISIBLE_CLASS);
-            this._attachedViewEvents.push(tooltip.on('clickoutside', Y.bind(this._handleClickOutside, this)));
         },
 
         /**
-         * Hides the tooltip
+         * Show the field description
          *
-         * @method _hideTooltip
+         * @method _showDescription
          * @protected
          */
-        _hideTooltip: function () {
-            var tooltip = this.get('container').one(TOOLTIP_SEL);
-
-            tooltip.removeClass(IS_VISIBLE_CLASS);
-            tooltip.removeClass(IS_DISPLAYED_CLASS);
-            tooltip.detach('clickoutside');
+        _showDescription: function () {
+            var container = this.get('container');
+            if (this._toggleFieldDefinitionDescription === 'active') {
+                if (this._isTouch() &&  container.one(TOOLTIP_DESCR)) {
+                    this._setTooltipPosition(FIELD_INPUT);
+                } else {
+                    container.addClass(IS_SHOWING_DESCRIPTION);
+                }
+            }
         },
 
         /**
-         * Considers tooltip's height and position on the screen to decide if it
-         * fits on the screen in current conditions
+         * Set the description tooltip position in Y axis.
+         * The tooltip position is modified by it's height.
          *
-         * @method _tooltipFitsTailUp
+         * @method _setTooltipPosition
          * @protected
-         * @return {boolean} true, if the tooltip fits in tail-up state
          */
-        _tooltipFitsTailUp: function () {
+        _setTooltipPosition: function (fieldInputDomClass) {
             var container = this.get('container'),
-                tooltip = container.one(TOOLTIP_SEL),
-                infoIcon = container.one(FIELD_INFO_ICON_SEL),
-                screenHeight = container.get('winHeight'),
-                scrollHeight = container.get('docScrollY'),
-                tooltipHeight = parseInt(tooltip.getComputedStyle('height'), 10),
-                infoIconHeight = parseInt(infoIcon.getComputedStyle('height'), 10);
-
-            return (infoIcon.getY() - scrollHeight + infoIconHeight + tooltipHeight < screenHeight);
-        },
+                tooltip = container.one(TOOLTIP_DESCR),
+                fieldInput = container.one(fieldInputDomClass),
+                tooltipHeight;
 
-        /**
-         * Event handler for a tap on the "close" link of a tooltip
-         *
-         * @method _handleCloseTooltipTap
-         * @param e {Object} Event facade object
-         * @protected
-         */
-        _handleCloseTooltipTap: function (e) {
-            e.preventDefault();
-            this._hideTooltip();
+            tooltip.addClass(IS_DISPLAYED_CLASS);
+            tooltipHeight = parseInt(tooltip.getComputedStyle('height'), 10);
+            tooltip.setY(fieldInput.getY() - tooltipHeight);
+            tooltip.addClass(IS_VISIBLE_CLASS);
```

Comment:
you add two classes on the tooltip with very similar semantic `IS_DISPLAYED_CLASS` and `IS_VISIBLE_CLASS`, that's odd, is that really needed ?